package.json

```diff
@@ -5,7 +5,7 @@
     "clean:generated": "find ./packages -name \"*.d.ts\" -type f -delete && find ./packages -name \"*.tsbuildinfo\" -type f -delete",
     "start": "yarn build -w",
     "start:example:js": "yarn build && npx http-server ./packages/js -o /example",
-    "lint": "eslint . --ext .ts,.tsx",
+    "lint": "eslint ./packages/**/src/*.{ts,tsx}",
```

Comment:
I think we should lint all the files in the repo. Furthermore, eslint now lints `ts` and `tsx` files by default, so no need to use specify that. I tested and when this PR https://github.com/Amsterdam/matomo-tracker/pull/197 is merged (because it removes `bundle.min.js` and `rollup.config.js`), you can add `es/` to the `.eslintignore` file and just use this: `"lint": "eslint ."`

Reply:
ok done,
but this should be merged after #197 as I didn't `exclude build.min.js`

I'm also wondering if it's not better to explicitly ignore `/packages/**/es/` and `/packages/**/lib/` in `.eslintignore`